### ISSUE TITLE
[Bug]: "hires. fix" prompt sharing same labels with txt2img_prompt

### ISSUE THREAD

**Issue by Linaqruf** [anapnoe/stable-diffusion-webui-ux#171]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

Hi thanks for your good work, however I noticed the problem in latest commits. It looks like "hires. fix" prompt sharing same labels with txt2img_prompt

Similar bug happened in [AUTOMATIC1111/stable-diffusion-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui) 2 weeks ago, it looks like the same bug occurs in your Web UI/UX. 

Somebody also reported this issue in my repo https://github.com/Linaqruf/sd-notebook-collection/issues/42. And I can confirm I got the same issue as well

<table>
  <tr>
    <th>Txt2img result</th>
    <th>Hi-res fix result</th>
  </tr>
  <tr>
    <td><img src="https://github.com/anapnoe/stable-diffusion-webui-ux/assets/50163983/135845a7-9a16-4b54-b015-37125daa8038" alt="Txt2img result" style="width:200px;height:150px;"></td>
    <td><img src="https://github.com/anapnoe/stable-diffusion-webui-ux/assets/50163983/75db10f5-945f-40fd-80e0-5ca49b502915" alt="Hi-res fix result" style="width:200px;height:150px;"></td>
  </tr>
</table>

I noticed "hires prompt" recorded in metadata even though i disabled hires prompt feature in settings. It looks like hires fix prompt stay there and not changed with the` txt2img_prompt`. Because I set default initial prompt "masterpiece, best quality, " to config.json, 

![image](https://github.com/anapnoe/stable-diffusion-webui-ux/assets/50163983/addc3d9e-62d9-4c3e-917d-b9c336ee1314)

### Steps to reproduce the problem

1. Go to https://colab.research.google.com/github/Linaqruf/sd-notebook-collection/blob/main/cagliostro-colab-ui.ipynb
2. Run cell `Install Cagliostro Colab UI` > `Download Model and VAE` > `Custom Download Corner` (this will download model for reproduce the problem from pastebin > `
Start Cagliostro Colab UI`
3. Use this prompt 
```
masterpiece, best quality,  1girl, solo, watercolor, birdwatching, nature reserve, 20s, green hair, pixie cut, cap, blue eyes, round eyes, curious expression, binoculars, casual clothes, field guide, trees, bushes, lake, birds, binocular strap, standing pose, looking through binoculars, observing birds.<lyco:hitokomoru_locon_new:0.7> <lyco:GoodHands-beta2:1.0>
```
5. and this seed : `3610680351`

Full Parameter:
```
parameters

masterpiece, best quality, 1girl, solo, watercolor, birdwatching, nature reserve, 20s, green hair, pixie cut, cap, blue eyes, round eyes, curious expression, binoculars, casual clothes, field guide, trees, bushes, lake, birds, binocular strap, standing pose, looking through binoculars, observing birds.<lyco:hitokomoru_locon_new:0.7> <lyco:GoodHands-beta2:1.0>
Negative prompt: (worst quality, low quality:1.4)
Steps: 20, Sampler: DPM++ 2M Karras, CFG scale: 7, Seed: 3610680351, Size: 512x768, Model: Amedira, Clip skip: 2
```

### What should have happened?

hires fix prompt doesn't shared the same label with txt2img_prompt.

### Commit where the problem happens

9931e861dfb128735c4a928a7beb5b5c0af30593

### What Python version are you running on ?

Python 3.10.x

### What platforms do you use to access the UI ?

Linux

### What device are you running WebUI on?

Other GPUs

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
Launching Web UI with arguments: --enable-insecure-extension-access --disable-safe-unpickle --xformers --multiple --share --no-hashing --disable-console-progressbars --opt-sub-quad-attention --opt-channelslast --no-download-sd-model --gradio-queue --listen --ckpt-dir=/content/cagliostro-colab-ui/models/Stable-diffusion --vae-dir=/content/cagliostro-colab-ui/models/VAE --hypernetwork-dir=/content/cagliostro-colab-ui/models/hypernetworks --embeddings-dir=/content/cagliostro-colab-ui/embeddings --lora-dir=/content/cagliostro-colab-ui/models/Lora --lyco-dir=/content/cagliostro-colab-ui/models/Lora --lowram --no-half-vae --theme dark
```

### List of extensions
```
[
    "Bing-su/adetailer",
    "ilian6806/stable-diffusion-webui-state",
    "klimaleksus/stable-diffusion-webui-fix-image-paste",
    "ashen-sensored/stable-diffusion-webui-two-shot",
    "hnmr293/sd-webui-cutoff",
    "camenduru/sd-webui-tunnels",
    "arenatemp/stable-diffusion-webui-model-toolkit",
    "bbc-mc/sdweb-merge-block-weighted-gui",
    "etherealxx/batchlinks-webui",
    "SignalFlagZ/sd-civitai-browser",
    "Zuellni/Stable-Diffusion-WebUI-Image-Filters",
    "Linaqruf/Umi-AI-debloat",
    "butaixianran/Stable-Diffusion-Webui-Civitai-Helper",
    "bbc-mc/sdweb-xyplus",
    "Coyote-A/ultimate-upscale-for-automatic1111",
    "hnmr293/sd-webui-llul",
    "kohya-ss/sd-webui-additional-networks",
    "Mikubill/sd-webui-controlnet",
    "opparco/stable-diffusion-webui-composable-lora",
    "pkuliyi2015/multidiffusion-upscaler-for-automatic1111",
    "zanllp/sd-webui-infinite-image-browsing",
    "derrian-distro/sd_webui_stealth_pnginfo",
    "KohakuBlueleaf/a1111-sd-webui-lycoris",
    "hako-mikan/sd-webui-regional-prompter",
    "thomasasfk/sd-webui-aspect-ratio-helper",
    "DominikDoom/a1111-sd-webui-tagcomplete",
    "mcmonkeyprojects/sd-dynamic-thresholding",
    "AlUlkesh/sd_delete_button"
]
```

### Console logs

```Shell
================================================================================
Launching 'anapnoe/stable-diffusion-webui-ux'
================================================================================
Selected VAE: Anime.vae.safetensors
================================================================================

fatal: No names found, cannot describe anything.
Python 3.10.12 (main, Jun  7 2023, 12:45:35) [GCC 9.4.0]
Version: <none>
Commit hash: 9931e861dfb128735c4a928a7beb5b5c0af30593
Installing requirements







Launching Web UI with arguments: --enable-insecure-extension-access --disable-safe-unpickle --xformers --multiple --share --no-hashing --disable-console-progressbars --opt-sub-quad-attention --opt-channelslast --no-download-sd-model --gradio-queue --listen --ckpt-dir=/content/cagliostro-colab-ui/models/Stable-diffusion --vae-dir=/content/cagliostro-colab-ui/models/VAE --hypernetwork-dir=/content/cagliostro-colab-ui/models/hypernetworks --embeddings-dir=/content/cagliostro-colab-ui/embeddings --lora-dir=/content/cagliostro-colab-ui/models/Lora --lyco-dir=/content/cagliostro-colab-ui/models/Lora --lowram --no-half-vae --theme dark
Civitai Helper: Get Custom Model Folder
Civitai Helper: Load setting from: /content/cagliostro-colab-ui/extensions/Stable-Diffusion-Webui-Civitai-Helper/setting.json
Civitai Helper: No setting file, use default
[-] ADetailer initialized. version: 23.6.2, num models: 8
[AddNet] Updating model hashes...
100% 6/6 [00:00<00:00,  7.73it/s]
[AddNet] Updating model hashes...
100% 6/6 [00:00<00:00, 8097.11it/s]
2023-06-11 02:22:20,307 - ControlNet - INFO - ControlNet v1.1.224
ControlNet preprocessor location: /content/cagliostro-colab-ui/extensions/sd-webui-controlnet/annotator/downloads
2023-06-11 02:22:20,493 - ControlNet - INFO - ControlNet v1.1.224
all detected, remote.moe trying to connect...
Warning: Permanently added 'localhost.run,35.171.254.69' (RSA) to the list of known hosts.
all detected, cloudflared trying to connect...
Download cloudflared...: 100% 35.1M/35.1M [00:00<00:00, 179MB/s]
Checkpoint AnyLoRA_Anime_Mix.safetensors not found; loading fallback Amedira.safetensors
Loading weights [None] from /content/cagliostro-colab-ui/models/Stable-diffusion/Amedira.safetensors
Creating model from config: /content/cagliostro-colab-ui/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading VAE weights specified in settings: /content/cagliostro-colab-ui/models/VAE/Anime.vae.safetensors
Applying optimization: xformers... done.
Textual inversion embeddings loaded(14): bad-artist, bad-artist-anime, bad-hands-5, bad-image-v2-39000, bad_prompt, bad_prompt_version2, BadDream, EasyNegative, EasyNegativeV2, FastNegativeEmbedding, FastNegativeEmbeddingStrong, ng_deepnegative_v1_75t, ngng00, UnrealisticDream
Textual inversion embeddings skipped(7): wdbadprompt, nartfixer, re-badprompt, nrealfixer, nfixer, rev2-badprompt, Mayng
Model loaded in 13.5s (load weights from disk: 8.2s, create model: 3.1s, apply weights to model: 0.1s, load VAE: 1.8s, load textual inversion embeddings: 0.2s).
Public WebUI Colab URL: None 
Public WebUI Colab URL: https://4dc25962-a90c-4698.gradio.live 
Public WebUI Colab URL: https://6db77601162dbe.lhr.life
Please do not use this link we are getting ERROR: Exception in ASGI application:  https://f9f51b3170f55d953a.gradio.live
Public WebUI Colab URL: https://deep-vulnerability-resource-mambo.trycloudflare.com
Startup time: 39.5s (import torch: 4.4s, import gradio: 0.9s, import ldm: 2.3s, other imports: 2.1s, setup codeformer: 0.2s, load scripts: 12.2s, create ui: 14.5s, gradio launch: 3.0s).
100% 20/20 [00:11<00:00,  1.75it/s]
100% 20/20 [00:04<00:00,  4.13it/s]
100% 20/20 [00:37<00:00,  1.89s/it]
100% 20/20 [00:07<00:00,  2.78it/s]
100% 20/20 [00:35<00:00,  1.77s/it]
```

### Additional information

The same issues already fixed in [AUTOMATIC1111/stable-diffusion-webui/](https://github.com/AUTOMATIC1111/stable-diffusion-webui/) 2 weeks ago with this PR https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/10785

And this is for comparison with [AUTOMATIC1111/stable-diffusion-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui) 

<table>
  <tr>
    <th>Txt2img result</th>
    <th>Hi-res fix result</th>
  </tr>
  <tr>
    <td><img src="https://github.com/anapnoe/stable-diffusion-webui-ux/assets/50163983/135845a7-9a16-4b54-b015-37125daa8038" alt="Txt2img result" style="width:200px;height:150px;"></td>
    <td><img src="https://github.com/anapnoe/stable-diffusion-webui-ux/assets/50163983/d9a3b868-876e-4b7f-8c07-5d6e0e5eb3d9" alt="Hi-res fix result" style="width:200px;height:150px;"></td>
  </tr>
</table>

**Comment by anapnoe** [comment]
fixed

**Comment by Linaqruf** [comment]
Thank you
